src/components/Search/SearchViewInit.tsx

```diff
@@ -1,21 +1,18 @@
 import React from 'react';
 
-import { connect } from 'react-redux';
+import { connect, useDispatch } from 'react-redux';
 
-import { store } from '../../common/store';
 import { fetchRecentRevisions } from '../../thunks/searchThunk';
 import { Repository, State } from '../../types/state';
 
 // component to fetch recent revisions when search view is loaded
-class SearchViewInit extends React.Component<SearchViewInitProps> {
-  componentDidMount() {
-    const { repository } = this.props;
-    store.dispatch(fetchRecentRevisions(repository));
```

Comment:
I was getting an error for 'promises must be awaited...etc', but the promise is awaited in `fetchRecentRevisions` with a `catch` block to catch errors.